Wikipedia/Code/WMFReferencePageViewController.swift

```diff
@@ -0,0 +1,108 @@
+
+import Foundation
+
+extension UIViewController {
+    class func wmf_viewControllerFromReferencePanelsStoryboard() -> Self {
```

Comment:
It's used to get different VC's from that storyboard - WMFReferencePanelViewController *or* WMFReferencePageViewController.